adapter/krw-usd.adapter.json

```diff
@@ -0,0 +1,74 @@
+{
+  "adapterHash": "0x0fb2222d8fc27dd7d25611054bbb43d1ba913ac218e1089724b7b0ab150f905a",
+  "name": "KRW-USD",
+  "decimals": 8,
+  "feeds": [
+    {
+      "name": "Dunamu-KRW-USD",
+      "definition": {
+        "url": "https://quotation-api-cdn.dunamu.com/v1/forex/recent?codes=FRX.KRWUSD",
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "method": "GET",
+        "reducers": [
+          { "function": "INDEX", "args": 0 },
+          { "function": "PARSE", "args": ["basePrice"] },
+          { "function": "DIVFROM", "args": 1 },
+          { "function": "POW10", "args": 8 },
+          { "function": "ROUND" }
+        ]
+      }
+    },
+    {
+      "name": "Yahoo-KRW-USD",
+      "definition": {
+        "url": "https://query1.finance.yahoo.com/v7/finance/spark?symbols=USDKRW%3DX",
+        "headers": { "Content-Type": "application/json" },
+        "method": "GET",
+        "reducers": [
+          { "function": "PARSE", "args": ["spark", "result"] },
+          { "function": "INDEX", "args": 0 },
+          { "function": "PARSE", "args": ["response"] },
+          { "function": "INDEX", "args": 0 },
+          { "function": "PARSE", "args": ["meta", "regularMarketPrice"] },
+          { "function": "DIVFROM", "args": 1 },
+          { "function": "POW10", "args": 8 },
+          { "function": "ROUND" }
+        ]
+      }
+    },
+    {
+      "name": "Naver-KRW-USD",
+      "definition": {
+        "url": "https://m.stock.naver.com/front-api/v1/marketIndex/productDetail?category=exchange&reutersCode=FX_USDKRW",
+        "headers": { "Content-Type": "application/json" },
+        "method": "GET",
+        "reducers": [
+          { "function": "PARSE", "args": ["result", "calcPrice"] },
+          { "function": "DIVFROM", "args": 1 },
+          { "function": "POW10", "args": 8 },
+          { "function": "ROUND" }
+        ]
+      }
+    },
+    {
+      "name": "Daum-KRW-USD",
+      "definition": {
+        "url": "https://finance.daum.net/api/exchanges/FRX.KRWUSD",
+        "headers": {
+          "Content-Type": "application/json",
+          "referer": "https://finance.daum.net/exchanges/FRX.KRWUSD",
+          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
```

Comment:
@nick-bisonai Do we have to specify the user agent to be able to access the data? 🤔

Reply:
yes, request is blocked without `User-Agenet` and `referer` headers :(